src/pages/Product/Product.js

```diff
@@ -0,0 +1,290 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import styled from 'styled-components';
+
+function Product() {
+  const [inputValue, setInputValue] = useState('');
+  const [commentList, setCommentList] = useState('');
+
+  useEffect(() => {
+    fetch('/data/comment.json')
+      .then(res => res.json())
+      .then(res => setCommentList(res.data.comment));
+  }, []);
+
+  const onChangeCommentInput = useCallback(e => {
+    setInputValue(e.target.value);
+  }, []);
+
+  const onSubmitComment = useCallback(e => {
+    // fetch(API), POST
+    // .then(res => res.json())
+    // .then(res => console.log(res))
+  }, []);
```

Comment:
추후 구현 예정이라서 주석처리하신건가요??
이런 경우에
```suggestion
  const onSubmitComment = useCallback(e => {
    // ToDo: 백엔드 API 완성 후 구체화 예정
    // fetch(API), POST
    // .then(res => res.json())
    // .then(res => console.log(res))
  }, []);
```
이런식으로 나중에 할거다라는 간단한 표시 남겨주면 알아보기도 쉽고, 나중에 전체검색에서 ToDo 등 키워드로 검색해서 해야하는 부분들 확인하기도 쉽습니다!